examples/gpgpu.rs

```diff
@@ -2,11 +2,16 @@
 extern crate glium;
 extern crate rand;
 use glium::glutin;
+use glutin::dpi::PhysicalSize;
 
 fn main() {
     let event_loop = glium::glutin::EventsLoop::new();
     let context_builder = glutin::ContextBuilder::new();
-    let context = glutin::Context::new(&event_loop, context_builder, false).unwrap();
+    let size = PhysicalSize {
+        width: 800.0,
+        height: 600.0,
+    };
+    let context = glutin::Context::new_headless(&event_loop, context_builder, size).unwrap();
```

Comment:
Nit: go `glutin::ContextBuilder::new().build_headless(...).unwrap()`, instead of `glutin::Context::new_headless(..).unwrap()`.